exercises/matrix/src/test/java/MatrixTest.java

```diff
@@ -16,7 +16,6 @@ public void extractRowFromOneNumberMatrixTest() {
         assertArrayEquals(expectedRow, matrix.getRow(rowIndex));
     }
 
-    @Ignore("Remove to run test")
     @Test
```

Comment:
Except for the first test case, please re-include the @Ignore annotation

Reply:
But would that not run the tests as per canonical-data.json as you're ignoring some tests?

Reply:
The annotation has no effect on testing with gradle (you can test with `gradle test`). See for example other PRs or just try for yourself. It is a track policy to include the annotation as explained.